### ISSUE TITLE
Style Transfer with Webcam - Video - Not working or very slow

### ISSUE THREAD

**Issue by palmierieugenio** [ml5js/ml5-examples#57]
I've tried to use the "Style Transfer with Webcam" demo both online and offline with the source code, but it's not working on my computer. It  loads the model, but when I press "start", it just freeze the camera. The online version on Chrome (not on Firefox) worked after waiting a lot, but it was still very laggy. Is it possible to make it faster and make the source code work?

**Comment by cvalenzuela** [comment]
How many FPS you get largely depends on your computer specifications. What are your specifications?

**Comment by palmierieugenio** [comment]
Yes, I didn't considered my computer, because it usually works well (I have tried also on my phone and the computer of a friend of mine, with same results: webcam freezed).

I expected it to not be computationally intensive, since it was part of the tutorials.

Here my specifications:
------------------
System Information
------------------

Operating System: Windows 7 Ultimate 64-bit (6.1, Build 7601) 
BIOS: Default System BIOS
Processor: Intel(R) Xeon(R) CPU X3470  @ 2.93GHz (8 CPUs), ~2.9GHz
Memory: 8192MB RAM

PS: I'd like to use ml5 also to make graphics for a musical project, so for me it would be important to be able to make the program run smoothly. Thanks for your work by the way, this project is really cool!

**Comment by cvalenzuela** [comment]
I'm not able to reproduce the problem. Any updates on your side?

**Comment by cvalenzuela** [comment]
yep, it is. Are you having the same issue?

**Comment by aagarwal1012** [comment]
@cvalenzuela, yes!
Actually I am running a 5 fps 100×100 video on a gaming laptop, then also I am facing the same issue as above.

**Comment by marksirvine** [comment]
At first I wasn't able to run the code locally, only after I ran a server using `cd ml5-examples` and `python -m SimpleHTTPServer` was I able to get it to run! 

Hope this helps.

**Comment by palmierieugenio** [comment]
> 
> 
> I'm not able to reproduce the problem. Any updates on your side?

No, I still have the same problem. I think it's not a problem if it needs some requirements: this functionality has a lot of potential and can be worth buying some hardware, but I think it would be useful to quantify them, having some suggested minimum requirements to make it run and requirements to make it run smoothly.

I think it could be useful also to understand if it's a hardware or software problem, since it seems it could be related to the kind of browser/server.
